services/api_gateway/src/main/java/com/futurewei/alcor/apigateway/filter/KeystoneConfiguration.java

```diff
@@ -34,14 +34,22 @@
     @Autowired
     private CacheFactory cacheFactory;
 
-    @Bean
+    //@Bean
     public KeystoneAuthWebFilter keystoneAuthWebFilter(){
         if(!keystoneEnable){
             return null;
         }
         return new KeystoneAuthWebFilter();
     }
 
+    @Bean
+    public KeystoneAuthGwFilter keystoneAuthGwFilter(){
+        if(!keystoneEnable){
+            return null;
+        }
+        return new KeystoneAuthGwFilter();
+    }
```

Comment:
duplicated issue or merge issue?

Reply:
They are different filter. KeystoneAuthGwFilter is used for Spring Gateway. KeystoneAuthWebFilter is used for Webflux filter.
Since webflux is now replaced with spring gateway, KeystoneAuthWebFilteris no longer useful。

Reply:
I see. Thanks for the clarification.